Pipfile

```diff
@@ -26,6 +26,7 @@ pytest-deadfixtures = "*"
 "flake8-string-format" = "*"
 "flake8-debugger" = "*"
 binaryornot = "*"
+pytest-picked = "*"
```

Comment:
You have moved `pytest-picked` into the wrong `Pipfile`.
Please, install `pytest-picked` into the `{{cookiecutter.project_name}}/Pipfile` only.

This should be removed. Also, make sure that `./Pipfile.lock` is also not changed.